app/organisation/sender.py

```diff
@@ -0,0 +1,34 @@
+from flask import current_app
+
+from app.config import QueueNames
+from app.dao.organisation_dao import dao_get_users_for_organisation
+from app.dao.services_dao import dao_fetch_service_by_id
+from app.models import EMAIL_TYPE, KEY_TYPE_NORMAL
+from app.notifications.process_notifications import (
+    persist_notification,
+    send_notification_to_queue,
+)
+
+
+def send_notification_to_organisation_users(
+    *, organisation, template, reply_to_text, personalisation=None, include_user_fields=None
+):
+    personalisation = personalisation or {}
+    include_user_fields = include_user_fields or []
+    active_users = dao_get_users_for_organisation(organisation.id)
```

Comment:
Just a question about how the feature will work - will it always be the case that any org user can approve requests? And likewise that you need to have a Notify account and be an org user in order to approve requests? I was wondering whether there might be org users who only want to view the billing, for example, who aren't interested in whether a service can go live or not

Reply:
Yes, for any organisation which has the `can_make_own_services_live` flag set to `true`. In the future we will probably need an organisation permission model to support the different levels of access you are talking about.